### PR TITLE
Save system tables from s3_disk in the report

### PR DESCRIPTION
**Pull request by tavplubix**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-clickhouse-ci-2** [comment]
This is an automated comment for commit d9438b176c249068aac433ee8b1e90ff2f816e50 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/54198/d9438b176c249068aac433ee8b1e90ff2f816e50/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

**Review by alexey-milovidov — APPROVED** [review]

File: docker/test/stateless/run.sh [hunk 1]
```diff
@@ -231,10 +232,16 @@ do
     fi
 done
 
+data_path_config="--path=/var/lib/clickhouse/"
+if [[ -n "$USE_S3_STORAGE_FOR_MERGE_TREE" ]] && [[ "$USE_S3_STORAGE_FOR_MERGE_TREE" -eq 1 ]]; then
+    # We need s3 storage configuration (but it's more likely that clickhouse-local will fail for some reason)
```
Comment (azat) [inline docker/test/stateless/run.sh:237]: Out of curiosity, which are they?

Comment (azat) [reply docker/test/stateless/run.sh:237]: @Felixoid interesting, after my comment in this PR I got this from CI / `MergeOnApproval`:

```
2023-11-22 20:03:14,680 User robot-ch-test-poll1 with 4807 remaining requests is used
Traceback (most recent call last):
  File "/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/tests/ci/merge_pr.py", line 294, in <module>
    main()
  File "/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/tests/ci/merge_pr.py", line 233, in main
    repo.organization._url.value = repo.organization.url.replace(  # type: ignore
AttributeError: can't set attribute 'value'
Error: Process completed with exit code 1.
```

Comment (Felixoid) [reply docker/test/stateless/run.sh:237]: it's fixed in master for a week

**Pull request merged**